<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>168</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>168</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otherExtLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pptCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ppt|pptx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pdfCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OtherheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>OtherheckBox</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>otherExtLineEdit</zorder>
   <zorder>pptCheckBox</zorder>
   <zorder>pdfCheckBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
